.github/workflows/build-docker.yaml

```diff
@@ -0,0 +1,54 @@
+name: Docker Build and Push
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          submodules: true
+      # - name: Login to Artifactory
```

Comment:
Should the commented code be removed ?

Reply:
Thanks, updated. Thought we might be able to add artifactory push as well, but will probably end up doing that separately